Traceback (most recent call last):
  File "get_vibrations.py", line 869, in <module>
    main()
  File "get_vibrations.py", line 523, in main
    data=open(folder+'/'+filename)
FileNotFoundError: [Errno 2] No such file or directory: 'WSe2.i_atom_38.i_coord_2.displ_0.0025/aims.out'
